src/utils.cairo

```diff
@@ -25,3 +27,38 @@ fn try_selector_with_fallback(
         }
     }
 }
+
+/// Manually deletes `erc165_id` value from the storage variable
+/// `ERC165_supported_interfaces` in the calling contract's storage.
+/// This function should only be used by a migration initializer during the
+/// ERC165-to-SRC5 migration process.
+fn deregister_erc165_interface(erc165_id: felt252) {
```

Comment:
Yeah, that could have worked as well. Since we were only going to delete the values, I thought a function would have been simpler both in design and usability